src/client/src/rt-platforms/defaultPlatformWindow.ts

```diff
@@ -0,0 +1,8 @@
+import { PlatformWindow } from './platformWindow'
+
+export function createDefaultPlatformWindow(window: Window): PlatformWindow {
```

Comment:
this is simple implementation of `PlatfromWindow` - it wraps web window and exposes basic methods